ledger/ledger-consensual/src/main/kotlin/net/corda/v5/ledger/consensual/ConsensualLedgerService.kt

```diff
@@ -53,23 +51,22 @@ interface ConsensualLedgerService {
      * @return The fully signed [ConsensualSignedTransaction] that was recorded.
      */
     @Suspendable
-    fun finality(
+    fun finalize(
         signedTransaction: ConsensualSignedTransaction,
         sessions: List<FlowSession>
     ): ConsensualSignedTransaction
 
-
     /**
      * Verifies, signs and records a [ConsensualSignedTransaction].
      *
      * @param session The [FlowSession] to receive the [ConsensualSignedTransaction] from.
-     * @param verifier Verifies the received [ConsensualSignedTransaction].
+     * @param checkTransaction Verifies the received [ConsensualSignedTransaction].
      *
      * @return The fully signed [ConsensualSignedTransaction] that was received and recorded.
      */
     @Suspendable
     fun receiveFinality(
         session: FlowSession,
-        verifier: ConsensualSignedTransactionVerifier
+        checkTransaction: ConsensualSignedTransactionChecker
```

Comment:
Agree with @lankydan this would be better as `checker`.

Reply:
I have updated this to checker.